docs/README_DATADOG.md

```diff
@@ -0,0 +1,160 @@
+# .NET Tracer for Datadog APM
+
+## Installation and Usage
+
+Please [read our documentation](https://docs.datadoghq.com/tracing/setup/dotnet) for instructions on setting up .NET tracing and details about supported frameworks.
+
+## Downloads
+Package|Download
+-|-
+Windows and Linux Installers|[See releases](https://github.com/DataDog/dd-trace-dotnet/releases)
+`Datadog.Trace`|[![Datadog.Trace](https://img.shields.io/nuget/vpre/Datadog.Trace.svg)](https://www.nuget.org/packages/Datadog.Trace)
+`Datadog.Trace.OpenTracing`|[![Datadog.Trace.OpenTracing](https://img.shields.io/nuget/vpre/Datadog.Trace.OpenTracing.svg)](https://www.nuget.org/packages/Datadog.Trace.OpenTracing)
+
+## Build Status on `master`
+
+Pipeline          | Build Status
+------------------|-------------
+Unit tests        | [![Build Status](https://dev.azure.com/datadoghq/dd-trace-dotnet/_apis/build/status/unit-tests?branchName=master)](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/latest?definitionId=28&branchName=master)
+Integration tests | [![Build Status](https://dev.azure.com/datadoghq/dd-trace-dotnet/_apis/build/status/integration-tests?branchName=master)](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/latest?definitionId=27&branchName=master)
+
+# Development
```

Comment:
Could you please move from here to a DEVELOPING.md file?

Reply:
I moved the whole Development section to developing.md